source/simulator/melt.cc

```diff
@@ -1180,7 +1265,12 @@ namespace aspect
       FEValues<dim> fe_values (this->get_mapping(),
                                this->get_fe(),
                                quadrature,
-                               update_quadrature_points | update_values);
+                               update_quadrature_points | update_gradients | update_values);
```

Comment:
check if you can remove the gradients and change `compute_material_model_input_values` argument to `false`

Reply:
I can set use_strain_rate to false, but we still need the gradients (as the material model inputs always compute the pressure gradients). 

Oh, and I changed the `compute_material_model_input_values` to `material_model_inputs.reinit`, as this is our new way to write this, correct?

Reply:
Ok then do that.

Yes I think it is.